### PR TITLE
rocksdb 5.18.3

### PR DESCRIPTION
**Pull request by DennisOSRM**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rocksdb.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def install
     ENV["PORTABLE"] = "1"
     ENV["DEBUG_LEVEL"] = "0"
     ENV["USE_RTTI"] = "1"
+    ENV["ROCKSDB_DISABLE_ALIGNED_NEW"] = "1"
```
Comment (lembacon) [inline Formula/rocksdb.rb]: If this only fails on Sierra, we probably want to write it as:

```ruby
ENV["ROCKSDB_DISABLE_ALIGNED_NEW"] = "1" if MacOS.version <= :sierra
```

Comment (DennisOSRM) [reply Formula/rocksdb.rb]: Excellent suggestion, thanks. Put it in and the formula built fine.

**Comment by lembacon** [comment]
Thanks @DennisOSRM!

**Comment by sgindin** [comment]
👍

**Pull request closed without merging**